dist/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/index.css" />
+    <title>Leaderboard</title>
+  <script defer src="main.js"></script></head>
+  <body>
+    <h1>Leaderboard</h1>
+    <div class="container">
+      <div class="players">
+        <div class="aside">
+          <h3>Recent score</h3>
+          <button class="refresh-btn">Refresh</button>
+        </div>
+        <ul class="clm-score">
+          <li>Name: 100</li>
+          <li>Name: 200</li>
+          <li>Name: 300</li>
+          <li>Name: 12</li>
+          <li>Name: 3</li>
+        </ul>
+      </div>
+
+      <div class="score-form">
+        <h3>Add score</h3>
+        <form>
+          <input
+            type="text"
+            name="player"
+            id="input-player"
+            placeholder="Your name"
+          />
+          <input
+            type="number"
+            name="score"
+            id="score"
+            placeholder="Your score"
+          />
+          <button type="submit" class="button">Submit</button>
```

Comment:
- [OPTIONAL] The submit button is right aligned to the `Add Score section`,  please it will be nice to fix that.